frontend/public/components/utils/resource-log.jsx

```diff
@@ -45,6 +50,23 @@ const LogControls = ({dropdown, onDownload, toggleFullscreen, isFullscreen, stat
       {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
     </div>
     <div className="co-toolbar__group co-toolbar__group--right">
+      {!_.isEmpty(podLogLinks) && _.map(_.sortBy(podLogLinks, 'metadata.name'), link => {
+        const namespace = resource.metadata.namespace;
+        const namespaceFilter = link.spec.namespaceFilter;
+        if (namespaceFilter && namespace.search(new RegExp(namespaceFilter, 'g')) === -1) {
```

Comment:
I agree with @benjaminapetersen that we should add a try/catch since invalid values are a runtime error:

```
new RegExp('[')
SyntaxError: unterminated character class debugger eval code:1
    <anonymous> debugger eval code:1
```

Something like

```ts
          if (namespaceFilter) {
            try {
              const namespaceRegExp = new RegExp(namespaceFilter, 'g');
              if (namespace.search(namespaceRegExp)) {
                return null;
              }
            } catch (e) {
              console.warn('invalid log link regex', namespaceFilter, e);
              return null;
            }
          }
```

Reply:
Added.